### ISSUE TITLE
`// swiftlint:disable:this` does not work if it placed on next line to `// swiftlint:disable:next`

### ISSUE THREAD

**Issue by norio-nomura** [realm/SwiftLint#1388]
Reproducing code:
```swift
// swiftlint:disable:next identifier_name
func ViolatingIdentifierName(){ // swiftlint:disable:this function_body_length opening_brace
    print("1")
    print("2")
    print("3")
    print("4")
    print("5")
    print("6")
    print("7")
    print("8")
    print("9")
    print("10")
    print("1")
    print("2")
    print("3")
    print("4")
    print("5")
    print("6")
    print("7")
    print("8")
    print("9")
    print("20")
    print("1")
    print("2")
    print("3")
    print("4")
    print("5")
    print("6")
    print("7")
    print("8")
    print("9")
    print("30")
    print("1")
    print("2")
    print("3")
    print("4")
    print("5")
    print("6")
    print("7")
    print("8")
    print("9")
    print("40")
    print("1")
}
```
`function_body_length` and `opening_brace` are detected:
```terminal.session
swiftlint lint --enable-all-rules --path test.swift
Linting Swift files at path test.swift
Linting 'test.swift' (1/1)
Valid Docs rule is disabled in Swift 2.3 and later as it is non functional.
Missing Docs rule is disabled in Swift 2.3 and later as it is non functional.
/…/swiftlint-test/test.swift:2:1: warning: Function Body Length Violation: Function body should span 40 lines or less excluding comments and whitespace: currently spans 41 lines (function_body_length)
/…/swiftlint-test/test.swift:2:30: warning: Opening Brace Spacing Violation: Opening braces should be preceded by a single space and on the same line as the declaration. (opening_brace)
Done linting! Found 2 violations, 0 serious in 1 file.
```
